roles/lib_openshift/src/doc/serviceaccount

```diff
@@ -0,0 +1,26 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+# TODO: create doc
```

Comment:
Sorry, my bad, creating doc now. I'll ping you when I get it done.

I meant to do this _before_ I created the PR.